RadarSDK/RadarLogBuffer.m

```diff
@@ -0,0 +1,80 @@
+//
+//  RadarLogBuffer.m
+//  RadarSDK
+//
+//  Copyright © 2021 Radar Labs, Inc. All rights reserved.
+//
+
+#import "Radar.h"
```

Comment:
do we need to import the main header file here (and in `RadarLogBuffer.h`)? doesn't seem like we're calling any fns on the public interface

Reply:
ah yes will remove